sagemaker-builder/packing-a-tensorflow-model/packing_your_tensorflow_model.ipynb

```diff
@@ -0,0 +1,783 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Packing your TensorFlow model\n",
+    "\n",
+    "With Amazon SageMaker, you can package your own algorithms that can then be trained and deployed in the SageMaker environment. This notebook guides you through an example using TensorFlow that shows you how to build a container for SageMaker and use it for training.\n",
+    "\n",
+    "## Permissions\n",
+    "\n",
+    "Running this notebook requires permissions in addition to the normal `SageMakerFullAccess` permissions. This is because it creates new repositories in Amazon ECR. The easiest way to add these permissions is simply to add the managed policy `AmazonEC2ContainerRegistryFullAccess` to the role that you used to start your notebook instance. There's no need to restart your notebook instance when you do this, the new permissions will be available immediately.\n",
+    "\n",
+    "## The example\n",
+    "\n",
+    "In this example we show how to package a custom TensorFlow algorithm with a Python example which works with the CIFAR-10 dataset: [CIFAR-10]: http://www.cs.toronto.edu/~kriz/cifar.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The parts of the algo\n",
+    "\n",
+    "The `src` directory has all the components you need to package the sample algorithm for Amazon SageMager:\n",
+    "\n",
+    "    └── src\n",
+    "        ├── cifar10.py\n",
```

Comment:
would it help reader understand better if these were called train.py and model.py?

Reply:
Good idea.